server/users.ts

```diff
@@ -226,6 +226,8 @@ export class Connection {
 	autojoins: string;
 	/** The last bot html page this connection requested, formatted as `${bot.id}-${pageid}` */
 	lastRequestedPage: string | null;
+	/** The current chat page the connection has open */
+	currentPage: string | null;
```

Comment:
This should probably be refactored out into its own PR.

Reply:
Oh, yeah, i meant to do that. Forgot. Will do right now.

Reply:
Removed.